src/components/datepicker/datePicker.scss

```diff
@@ -163,3 +163,17 @@ md-datepicker[disabled] {
     display: none;
   }
 }
+
+// Animate the calendar inside of the floating calendar pane such that it appears to "scroll" into
+// view while the pane is opening. This is done as a cue to users that the calendar is scrollable.
+.md-datepicker-calendar-pane {
+  .md-calendar {
+    transform: translateY(150px);
+    transition: transform 0.4s $swift-ease-out-timing-function;
```

Comment:
Should `transition: ....` be in here instead:

```css
&.md-pane-open .md-calendar {
    transition: transform 0.4s $swift-ease-out-timing-function;
    transform: translateY(0);
}
```

to set the duration only when opening... but `0` duration when **not** opening.